### PR TITLE
fix: update admission rules for 2024/2025

### LINKED ISSUE #114: Call for admission 2024/2025
To compliant our TOL simulator with the call of admission of Politecnico di Milano, a.y. 2024/2025.

The algorithm for both calculating the P final score and OFA test needs to be modified:

- if P < 29.50 then TOL failed
- if P >= 74.50 then TOL passed
- if 29.50 <= P < 74.50 then TOL passed IFF student of the last year of high school or already graduated. We could always put "TOL passed", adding the previous notice at the end of the simulation, section "How the score is calculated".
- OFA Tests are no longer scheduled. The OFA test details in the “How the Score is Calculated” section should also be deleted.

[BandoLaureaIngegneria_20242025.pdf](https://github.com/PoliNetworkOrg/TheTOLProject/files/14268530/BandoLaureaIngegneria_20242025.pdf)

### PR DESCRIPTION
**Pull request by toto04**
closes #114
I've added another threshold to distinguish between normal passing and passing for the Early TOL.
the result message is the same as before for scores above 75, while it specifies passage for only standard rankings for scores between 30 and 75. mentions to the OFA TEST were removed and OFA TENG message is now the same for all possible results.

in the description all mentions of OFA TEST were removed
I've tried to describe how the thresholds work, trying to emphasize the difference between the right of enrollment in the Early TOL and the access to the rankings for the Standard TOL.

I've also put the link to the rankings website right after the explanation of the threshold for the standard rankings, the links points directly to the engineering page, not to the home page.
I've taken inspiration for how the tg links work in the locales files for the URL, I've put the domain as the label but that can be easily changed if anyone has anything better in mind.

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -63,7 +63,9 @@ export const DSATimeModifier = 1.3
 
 export const testTotalScore = 100
 /** Minimum score to pass the test */
-export const testPassThreshold = 60
+export const standardTestPassThreshold = 30
+/** Minimun score to pass the test on the fourth year of high school */
+export const earlyTestPassThreshold = 75
```
Comment (EndBug) [inline src/utils/constants.ts:68]: Quick question: why is it 30/75 when the original comment on the issue talks about 29,50/74,50?

Comment (zagbc) [reply src/utils/constants.ts:68]: Please note that according to PoliMi rules the total score must be rounded to the nearest integer (e.g. 29.49 is rounded to 29 while 74.50 is rounded to 75).

Comment (toto04) [reply src/utils/constants.ts:68]: > Please note that according to PoliMi rules the total score must be rounded to the nearest integer (e.g. 29.49 is rounded to 29 while 74.50 is rounded to 75).

This is exactly the reason, i kept the logic unchanged, checks are done with the score rounded to the nearest integer (before this, threshold was 60, and as far as i know a score of say 59.75 was rightly regarded as passing)

Comment (EndBug) [reply src/utils/constants.ts:68]: makes sense ✅

**Review by EndBug — APPROVED** [review]

**Pull request merged**